Workiva/vs-code-format-on-save
# Remove dartfmt

## Motivation

We just adopt Dart 2.18 and with that, `dartfmt` is no longer the formatting tool. This PR updates the executable to be "dart format" when the code path is taken.

## Changes

- Switch from `darfmt` executable to `dart format`
- Rebuild the extension v1.0.2 extension

## QA'ing
- [ ] Install the package and verify it works with Dart 2.18 in a repo without OverReact Format. You can just remove the OverReact Format dependency from the pubspec in any repo and restart VS Code to get that effect!
    - Clone this repo
    - In the repo, run `code --install-extension over-react-format-on-save-1.0.2.vsix`
    - Your VS Code OverReach Format on Save extension should be v1.0.2 now. 
        - If it's not, you can run remove the v1.0.1 (or whatever version) from the extensions directory (`~/.vscode/extensions`), and it your IDE should use v1.0.2
    - Open a project in VS Code, being sure you're using Dart 2.18
    - Verify formatting works as expected 
- [ ] Other details to check.. 
    - [ ] Logging is updated correct (uses "dart format" instead "dartfmt")
